Tools/Scripts/hooks/pre-push

```diff
@@ -294,8 +294,9 @@ def main(name, remote):
             return 255
         name = remote_to_name.get('{}:{}'.format(match.group('host'), match.group('path')), None)
         if not name:
+            name = 'user-specified'
```

Comment:
Is there any reason why we need this? This doesn't appear to some special known-value that appears elsewhere, can we not just keep using the specified URL?

Reply:
No, that's a good point. We can use the remote. Will update.

Reply:
(Pedantic terminology point: `git-push(1)` claims `<repository>` is either a URL or a remote, and we're likely in the former case at this point.)